test/test_gradients.jl

```diff
@@ -68,22 +65,21 @@ end
 	isic, dft, optchk = rand([true, false], 3)
 	optchk = optchk && !dft
     @timeit TIMEROUTPUT "LSRTM validity (isic=$(isic), checkpointing=$(optchk), dft=$(dft))" begin
-		ftol = fs ? 1f-3 : 5f-4
+		ftol = 5f-3
```

Comment:
`dt` is now runtime rather than `subs` so leads to slight increase of numerical roundoff errors.  Still fairly ok I think

Reply:
EDIT: was a tiny bug in checkpointing setup, reverted to previous tolerance. The one below is because of `dt` shows up more for such larger values (full wsvefield)